File: app/src/main/java/mb00/android/codehub/ui/repo/inject/RepoContributorsFragmentComponent.java
package mb00.android.codehub.ui.repo.inject;

import dagger.Subcomponent;
import dagger.android.AndroidInjector;
import mb00.android.codehub.ui.repo.view.RepoContributorsFragment;


@Subcomponent(modules = RepoContributorsFragmentModule.class)
public interface RepoContributorsFragmentComponent extends AndroidInjector<RepoContributorsFragment> {

    @Subcomponent.Builder
    abstract class Builder extends AndroidInjector.Builder<RepoContributorsFragment> {}

}

File: app/src/main/java/mb00/android/codehub/ui/repo/inject/RepoContributorsFragmentComponent.kt
package mb00.android.codehub.ui.repo.inject

import dagger.Subcomponent
import dagger.android.AndroidInjector
import mb00.android.codehub.ui.repo.view.RepoContributorsFragment


@Subcomponent(modules = [
    RepoContributorsFragmentModule::class
])
interface RepoContributorsFragmentComponent : AndroidInjector<RepoContributorsFragment> {

    @Subcomponent.Builder
    abstract class Builder : AndroidInjector.Builder<RepoContributorsFragment>()

}
